src/main/webapp/index.html

```diff
@@ -1,23 +1,30 @@
 <!DOCTYPE html>
-<html>
-  <head>
+<html lang="en">
+<head>
     <meta charset="UTF-8">
-    <title>Project</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-    <script src="https://accounts.google.com/gsi/client" async defer></script>
-    <script src="https://unpkg.com/jwt-decode/build/jwt-decode.js"></script>
-  </head>
-  <body>
-    <div id="login">
```

Comment:
Can you hold off from deleting this for now. I think we should output this information in the console, so that we know the API is functioning correctly. Taylor can work on this. Maybe add a TODO for Taylor to remove this?